_drafts/coop-values.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Coop Community Values"
+author: Lauren Wolfe
+categories: # delete the categories that do not apply and keep at least one
+  - Community
+tags:
+  - community values
+---
+
+Having gone through a few [transitions]() in recent months along with global and local conversations surrounding the COVID-19 pandemic and rampant anti-blackness in the United States have caused those of us at the Coop to take a step back and really consider why we do what we do. From these discussions we have seen a need for and are beginning to develop the Coop's Community Values. 
+
+For full transparency and to ensure we capture the actual values of our community members we will be holding a community conversation in the next week where community members can bring their thoughts and suggestions to the table. Following this we will formalize our values statements and release a document for public comment.
+
+## Why Develop Community Values?
+
+Community values are a direct response to the theme of individualism that is prevelant in American and academic culture. Individualism is a narrative that celebrates the successes of the individual over the common good shifts the blame for falling behind to the individual rather than examining root causes and systematic barriers to success. Policy rooted in individualism favor solutions that leave individual people to figure out solutions on their own rather than addressing challenges as a collective. 
+
+Prior to the Coop's existance, the data science policy at Fred Hutch was very much "you're on your own". Unless a lab group specifically built out their own onboarding tools, newcomers to Fred Hutch were left to their own devices when it came to finding information about data science infrastructure, best practices, and community groups. This way of doing things is not only largly inefficient but also ignores the systematic barriers that a person might face to gaining access to this information on their own. 
```

Comment:
existance -> existence
I personally feel like this paragraph should be softened, and combined with the following paragraph. I prefer to focus on how we could do things better as aligned with our values, as opposed to pointing out our flaws. This is especially true because I don't think we're actually doing better with onboarding now. What we want to do is make information accessible (and the reasons you make are very valid!), and our values will help us do that.